third_party/rust/Cargo.toml

```diff
@@ -12,6 +12,7 @@ members = ["cxxbridge_cmd/v1/crate"]
 anyhow = "1"
 async-trait = "0.1"
 base64 = "0.13"
+bech32 = "0.9.1"
```

Comment:
Please add the same line to `components/brave_wallet/rust/Cargo.toml` so future automation can re-generate the list correctly.

Reply:
Thanks for fixing!